src/metrics/rules/mapping.go

```diff
@@ -47,6 +50,9 @@ var (
 	errMappingRuleSnapshotIndexOutOfRange                  = errors.New("mapping rule snapshot index out of range")
 	errNilMappingRuleSnapshotProto                         = errors.New("nil mapping rule snapshot proto")
 	errNilMappingRuleProto                                 = errors.New("nil mapping rule proto")
+
+	m3MetricsGraphitePrefix = []byte("__m3_graphite_prefix__")
```

Comment:
Can you reference the declaration in the other package so we have this string declared in one place? Either that or move the string declaration to here and reference it from the other package (if the imports don't worry the first way).

Reply:
ah yes. its not exported there. let me see where best to put this..